lib/core/jhipster/reserved_keywords.js

```diff
@@ -207,7 +212,7 @@ function isReservedTableName(keyword, databaseType) {
 }
 
 function isReservedFieldName(keyword) {
-  return isReserved(keyword, 'ANGULAR') || isReserved(keyword, 'TYPESCRIPT') || isReserved(keyword, 'JAVA');
+  return isReserved(keyword, 'ANGULAR') || isReserved(keyword, 'TYPESCRIPT') || isReserved(keyword, 'JAVA') || isReserved(keyword, 'KOTLIN');
```

Comment:
I dont think we should check every blueprint language supported in JH core, this should be handled by the blueprint itself else this defeats the purpose of having independent blueprint